SwiftSyntaxDevUtils/Sources/swift-syntax-dev-utils/commands/Build.swift

```diff
@@ -11,29 +11,53 @@
 //===----------------------------------------------------------------------===//
 
 import ArgumentParser
-import Foundation
 
-struct Build: ParsableCommand, BuildCommand {
-  static var configuration: CommandConfiguration {
-    return CommandConfiguration(
-      abstract: "Build swift-syntax."
-    )
-  }
+struct Build: ParsableCommand {
+  static let configuration = CommandConfiguration(
+    abstract: "Build swift-syntax."
+  )
 
   @OptionGroup
   var arguments: BuildArguments
 
+  private var executor: Executor {
+    Executor(
+      compilationCoordinator: CompilationCoordinator(
+        toolchain: arguments.toolchain,
+        buildDir: arguments.buildDir,
+        multirootDataFile: arguments.multirootDataFile,
+        release: arguments.release,
+        enableRawSyntaxValidation: arguments.enableRawSyntaxValidation,
+        enableTestFuzzing: arguments.enableTestFuzzing,
+        warningsAsErrors: arguments.warningsAsErrors,
+        verbose: arguments.verbose
+      )
+    )
+  }
```

Comment:
I believe we can convert them to stored properties. The only reason I haven't done so was to avoid having to mark the `run` methods with the `mutable` keyword.

Reply:
What about changing the `run` methods to 

```swift
  func run() throws {
    let executor = Executor(
      swiftPMBuilder: SwiftPMBuilder(
        toolchain: arguments.toolchain,
        buildDir: arguments.buildDir,
        multirootDataFile: arguments.multirootDataFile,
        release: arguments.release,
        enableRawSyntaxValidation: arguments.enableRawSyntaxValidation,
        enableTestFuzzing: arguments.enableTestFuzzing,
        warningsAsErrors: arguments.warningsAsErrors,
        verbose: arguments.verbose
      )
    )
    try executor.run()
  }
```

Reply:
Local variable? Sound good to me–done.